src/components/Shared/UI/DTESelect/DTESelect.tsx

```diff
@@ -28,6 +28,21 @@ const StyledSelect = styled(Select)`
     }
   }
 `;
+
+export const capitaliseWords = (input: string): string => {
+  return input
+    .toLowerCase()
+    .split(" ")
+    .map((word) => {
+      if (/\d/.test(word)) {
```

Comment:
export const capitaliseWords = (input: string): string => {
  return input
    .toLowerCase()
    .split(" ")
    .map((word) => word.charAt(0).toUpperCase() + word.slice(1))
    .join(" ");
}; in this function i set everything to lowercase because, finding the css property that was causing the initial problem of turning the whole address into block capitals, was pretty much impossible because of priority, and because styling was imported from somewhere else, so now the entire address is lower case, the post code was too, like this e47tq, so i checked for digits and turned only that string to uppercase entirely, then capitalised the first letter of every word for those words with no numbers, to get a full address formatted with, the correct specifications. But the function above dosent handle the post code properly, my original function does Its a complicated work around